src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
+    window.open(`products/${searchItem}`)
+    setSearch("")
+  }
+
+  const handleKeyPress = (e) => {
+    if (e.key === 'Enter') {
+      handleClickSearch()
+    }
+  }
+
+  return (
+    <div className="home-search-box">
+      <input
+        value={searchItem}
+        type="text"
+        onChange={(e) => setSearch(e.target.value)}
+        onKeyUp={handleKeyPress}
+        placeholder={t('searchPlaceholder')}
+        name="search"
+      />
+      <button data-key="13" onClick={handleClickSearch} className="home-search-box__button" />
```

Comment:
Please disable this button when user input is empty. 
I have a question, do we really need this `data-key`?

```suggestion
      <button disabled={searchItem.trim()}  data-key="13" onClick={handleClickSearch} className="home-search-box__button" />
```

Reply:
Thanks for the info, I'll get up to apply input validation.
For the question: No, we don't.

Reply:
After I tried using this: `<button disabled={searchItem.trim()}  onClick={handleClickSearch} className="home-search-box__button" />`

the button is disabled when I input something, but clickable when the input is empty.